runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -406,6 +418,12 @@ public void keyReleased(KeyEvent e)
 		{
 			inOverlayDraggingMode = false;
 		}
+
+		if (!e.isShiftDown())
+		{
+			inMenuEntryMode = false;
+			menuEntries = null;
```

Comment:
This is weird since it would clear the menu just from releasing a key? Also this is on a different thread from the render thread so it might race with onClientTick

Reply:
Yeah i dont think its necessary, I will remove it